data/sponsors.js

```diff
@@ -0,0 +1,43 @@
+export const SPONSORS = {
+  platinumSponsors: [
```

Comment:
nit: Sponsors inside the object is duplicate. E.g: platinumSponsors can be just platinum.